host/xtest/pkcs11_1000.c

```diff
@@ -8415,6 +8443,12 @@ static CK_RV test_rsa_aes_wrap(ADBG_Case_t *c, CK_SESSION_HANDLE session,
 		  t->key.prime1_len },
 		{ CKA_PRIME_2, (CK_VOID_PTR)t->key.prime2,
 		  t->key.prime2_len },
+		{ CKA_EXPONENT_1, (CK_VOID_PTR)t->key.exp1,
+		  t->key.prime2_len },
+		{ CKA_EXPONENT_2, (CK_VOID_PTR)t->key.exp2,
+		  t->key.prime2_len },
+		{ CKA_COEFFICIENT, (CK_VOID_PTR)t->key.coeff,
+		  t->key.coeff_len },
```

Comment:
I don't think this information are needed. They can be recomputed by the pkcs11 token based on the provided `CKA_PRIVATE_EXPONENT`, `CKA_PRIME_1` and `CKA_PRIME_2` attributes.